app/views/catalog/_media_image.html.erb

```diff
@@ -18,7 +18,12 @@
       <% if document.file_sets.find { |fs| fs.id == fs_id }.pdf? %>
         <%= link_to(image_tag(img, style: 'width: 100%'), download_url(fs_id, only_path: true)) %>
       <% else %>
-        <%= image_tag(img, style: 'width: 100%') %>
+        <div class="row zoom">
```

Comment:
What does this look like when there are multiple images for an object, and the JavaScript carousel code takes effect?